framework/src/modules/http_api/controllers/signatures.js

```diff
@@ -73,12 +69,9 @@ SignaturesController.postSignature = async function(context, next) {
 		}
 	} catch (err) {
 		error = new ApiError(err, apiCodes.PROCESSING_ERROR);
-		console.log('inside');
-
-		context.statusCode = error.code;
-		delete error.code;
 	}
 
+	context.statusCode = error.code;
 	return next(error);
```

Comment:
why is line 79 https://github.com/LiskHQ/lisk/pull/3032/commits/f26c3ebfeecc68f9e0008027cbc5b0fe791ccd72#diff-85d54af64d1362a64fb0e78e4c17ce65L79 removed?

Reply:
Did it to have consistent behavior with transactions controller and write better tests using the `api_codes`. No breaking change.